### ISSUE TITLE
Exception - Match is not defined

### ISSUE THREAD

**Issue by brunolemos** [msavin/Mongol#106]
I keep getting this on server console:

``` sh
I20150908-18:53:48.390(-3)? Exception from sub MeteorToys id pQi2BmA4eAWaiL6dd ReferenceError: Match is not defined
I20150908-18:53:48.391(-3)?     at [object Object]._0x3f8a [as _handler] (packages/meteortoys_toykit/packages/meteortoys_toykit.js:9:1)
I20150908-18:53:48.392(-3)?     at maybeAuditArgumentChecks (livedata_server.js:1692:12)
I20150908-18:53:48.392(-3)?     at [object Object]._.extend._runHandler (livedata_server.js:1023:17)
I20150908-18:53:48.392(-3)?     at [object Object]._.extend._startSubscription (livedata_server.js:842:9)
I20150908-18:53:48.392(-3)?     at [object Object]._.extend.protocol_handlers.sub (livedata_server.js:614:12)
I20150908-18:53:48.392(-3)?     at livedata_server.js:548:43
```

**Comment by bwhitty** [comment]
I am also getting the same error. Running `meteor@1.2-rc.14`.

**Comment by brunolemos** [comment]
I'm also running v1.2, maybe it's related to that.

**Comment by brunolemos** [comment]
Ok, current solution:

``` sh
meteor add check
```

Now this package must be added to the dependency list.

**Comment by brunolemos** [comment]
Pull request submitted.

**Comment by msavin** [comment]
Fixed in Mongol v1.2

**Comment by AlABarazi** [comment]
I got this error when I used doctorpangloss:filter-collections package with meteor 1.3. 
meteor add check 
solved it.

Thanks brunolemos!
